client/block.c

```diff
@@ -1038,6 +1040,11 @@ static void *work_thread(void *arg)
 	int n_mining_threads = (int)(unsigned)(uintptr_t)arg, sync_thread_running = 0;
 	uint64_t nhashes0 = 0, nhashes = 0;
 	pthread_t th;
+	if(USE_ORPHAN_HASHTABLE){
```

Comment:
This macro can be removed since it's not used in the rest of code changes.

Reply:
But here is used ! :)

Reply:
If USE_ORPHAN_HASHTABLE equals 0, what will happen?

Reply:
It will use the original algorithm :D